pkg/policy/validate.go

```diff
@@ -1004,6 +1004,16 @@ func validateConditionValues(c kyverno.Condition) (string, error) {
 	if c.Key == nil || c.Value == nil || c.Operator == "" {
 		return "", fmt.Errorf("entered value of `key`, `value` or `operator` is missing or misspelled")
 	}
+	switch reflect.TypeOf(c.Key).Kind() {
+	case reflect.String:
```

Comment:
What about arrays? We need to validate list items as well.

Reply:
This function is to validating  value for "{{request.operation}} " variable . It not make sense to add {{request.operation}} is in array as value of variable will be single value . So not added for array.